alephium/test/token-bridge-test.ts

```diff
@@ -351,34 +359,35 @@ describe('test token bridge', () => {
     const tokenPoolState = testResult.contracts.filter(
       (c) => c.contractId === fixture.wrappedAlphPoolInfo.contractId
     )[0]
-    expect(tokenPoolState.fields['totalBridged']).toEqual(fixture.totalBridged + transferAmount)
+    const realTransferAmount = transferAmount - messageFee
```

Comment:
Do we test both zero fee and non-zero fee?

Reply:
Now only non-zero fee, will add tests for zero fee, thanks!